src/twisted/web/newsfragments/.gitignore

```diff
@@ -0,0 +1 @@
+!.gitignore
```

Comment:
I think that a better fix would be to have a README file which will explain what developers should put into it and the basic rules... maybe just a link to the documentation